src/vcf/record.cpp

```diff
@@ -318,7 +318,7 @@ namespace ebi
             try {
                 check_no_duplicates(format);
             } catch (const std::invalid_argument &ex) {
-                throw new FormatBodyError{line, "FORMAT must not have duplicate fields", ex.what()};
+                throw new FormatBodyError{line, "FORMAT must not have duplicate fields"};
```

Comment:
I was thinking about this one. we are not fixing this in the debugulator, but could we? I guess it's a bit difficult because we have to look at all the samples to see if the duplicated values match or not, and leave one field in each sample or none, respectively